### ISSUE TITLE
Train issues (loss = NaN, accuracy no changes)

### ISSUE THREAD

**Issue by junyongyou** [butterfliesss/SDT#15]
I have used the same script, same data produced by the authors, while cannot reproduce the results at all. The train loss is NaN from the beginning until the end. See:

epoch: 1, train_loss: nan, train_acc: 9.83, train_fscore: 4.84, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 7.26 sec
epoch: 2, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 2.59 sec
epoch: 3, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 2.12 sec
epoch: 4, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.89 sec
epoch: 5, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.87 sec

...

epoch: 143, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.86 sec
epoch: 144, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.99 sec
epoch: 145, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.85 sec
epoch: 146, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.84 sec
epoch: 147, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.82 sec
epoch: 148, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.93 sec
epoch: 149, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.87 sec
epoch: 150, train_loss: nan, train_acc: 8.67, train_fscore: 1.38, valid_loss: nan, valid_acc: nan, valid_fscore: nan, test_loss: nan, test_acc: 8.87, test_fscore: 1.45, time: 1.93 sec

Is there anyone who can reproduce the results in the paper?

**Comment by zzrobot1979** [comment]
Nobody

**Comment by liuying2023912** [comment]
Have you solved this problem here?
